src/Libraries/FIX/Source/fix.c

```diff
@@ -702,7 +702,7 @@ AWide *AsmWideBitShift(AWide *src, long shift)
 	return src;
 }
 
-AWide *WideSquareRoot(AWide *src, long div)
+AWide *WideSquareRoot(AWide *src)
 {
 	printf("WideSquareRoot\n");
```

Comment:
Maybe for these we should printf both of the variables as to not remove the params?